main/CMakeLists.txt

```diff
@@ -161,6 +158,7 @@ if (MINGW)
    install( FILES
       ${MINGW_ROOT}/bin/libstdc++-6.dll
       ${MINGW_ROOT}/bin/libwinpthread-1.dll
+      ${MINGW_ROOT}/bin/zlib1.dll
```

Comment:
This doesn't seem needed, not for me, not the a 32-bit build?

Reply:
I tried to run MuseScore after setting PATH to an empty string, and it complained about a missing zlib1.dll (as well as portaudio.dll because of the renaming). Indeed, MuseScore is linking against z library ( https://github.com/musescore/MuseScore/blob/a5de0cec8b594611ff1a9c0c760a0e345673fc73/mscore/CMakeLists.txt#L564 ); in the msvc case it is included as a static library shipped in the dependencies bundle, while in the MinGW case it should be searching for zlib dll.
Can you please try to run MuseScore after `set PATH=` ?

Reply:
Ah, PATH indeed is set to include ${MINGW_ROOT}/bin for me, so yes, this got to be why it isn't missing for me. But even if I

   set PATH=
   MuseScore3.exe

I don't get any errors and it just starts

Reply:
Even if you delete zlib1.dll from MuseScore's bin folder?

Reply:
It's not there

Reply:
Then it's some sort of mystery, or the usual Windows compilation "black magic".
If I check with Dependency Walker, it is searching for zlib1.dll, as I would expect since MuseScore3.exe is linked against z library.
I am using gcc 9.2, Qt 5.12.6, 64bit, under Windows 10

Reply:
I'm using Qt 5.12.6 and GCC 7.3 on Windows 7, and a 32bit build currently, but anyway, adding that dll surely won't harm. Will fire up a 64bit build on Windows 10 to doublecheck

Reply:
OK, I've now tried with Q 5.122.6 64bit, GCC 7.3 on Windows 10: still doesn't seem to need that dll

Reply:
Boh, still a mystery. Can you please try to analyze the executable MuseScore3.exe with Dependency Walker? http://www.dependencywalker.com/
Including it doesn't hurt, but I would prefer not including unneeded libraries and I find it strange that it is needed only in my pc. 😕

Reply:
It complains about 5 different API-MS-WIN-WORE-WINRT-*L1-1-0.DLL, DCOMP.DLL and IESHIMS.DLL, bit nothing else

Reply:
API-MS-WIN-CORE-WINRT-ERROR-L1-1-0.DLL
API-MS-WIN-CORE-WINRT-L1-1-0.DLL
API-MS-WIN-CORE-WINRT-ROBUFFER-L1-1-0.DLL
API-MS-WIN-CORE-WINRT-STRING-L1-1-0.DLL
DCOMP.DLL
IESHIMS.DLL

Reply:
It is more about the dependencies it found than what it does not find. Here is the screenshot of Dependency Walker for the build in my system: one of the dependency is zlib1.dll.
![Capture1](https://user-images.githubusercontent.com/2855332/71190332-b2dbd400-2284-11ea-9e37-15a24cb55643.png)

Reply:
No ZTLIB1.DLL
![image](https://user-images.githubusercontent.com/1786669/71192400-790ccc80-2288-11ea-90e9-7393215929cc.png)

Reply:
Found the reason why it is needed for your build? Related to the different MinGW version, maybe?

Reply:
Actually, I didn't find the real reason. If I use the MinGW-w64 shipped by Qt, the zlib dll is not needed (and indeed including it in the install step results in a "not found" error). If I use MinGW-w64 from msys64, then this dll is required.
I decided to remove this dll install step so that the compilation works out-of-the-box with the MinGW-w64 given by Qt installation.
So, in the end, I think this PR is ready to be merged.

Reply:
So including that dll would even cause harm, for those that use the Qt MinGW